src/helix-ui/styles/components/drawer.less

```diff
@@ -0,0 +1,39 @@
+@import (reference) 'vars';
+
+hx-drawer {
+  > * {
+    grid-area: body;
+    flex-grow: 1;
+    flex-shrink: 1;
+    order: 2;
+  }
+
+  > header {
+    grid-area: head;
+    flex-grow: 0;
+    flex-shrink: 0;
+    margin: 0 !important; // FIXME
+    order: 1;
+    padding: 0.75rem;
+    padding-right: 2.75rem;
+
+    > * {
+      font-size: 1.25rem;
+    }
+  }
+
+  > hx-div {
+    background-color: @gray-0;
+    border-top: 1px solid @gray-400;
+  }
+
+  > footer {
+    background-color: @gray-0;
+    border-top: 1px solid @gray-400;
+    grid-area: foot;
+    flex-grow: 0;
+    flex-shrink: 0;
+    order: 3;
+    padding: 0.75rem;
+  }
+}
```

Comment:
add overrides for polyfilled browsers

Reply:
override for polyfilled browswers added:

```css
#hxDrawer.hx-drawer {
  &:extend(hx-drawer all);
}
```